### PR TITLE
Allow extra stderr output from COCI and testlib checkers

### PR DESCRIPTION
**Pull request by kiritofeng**
(Tested against https://github.com/DMOJ/judge-testsuite/pull/58)

### CODE REVIEW

File: dmoj/contrib/coci.py [hunk 1]
```diff
@@ -1,26 +1,26 @@
-import re
```
Comment (quantum5) [inline dmoj/contrib/coci.py:1]: Why is the entire file changed?

File: dmoj/contrib/coci.py [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+import re
+
+from dmoj.contrib.testlib import ContribModule as TestlibContribModule
+from dmoj.error import InternalError
+from dmoj.result import CheckerResult
+
+
+class ContribModule(TestlibContribModule):
+    name = 'coci'
+    repartial = re.compile(br'^partial ((\d+)\/(\d*[1-9]\d*))\n$', re.M)
```
Comment (quantum5) [inline dmoj/contrib/coci.py:10]: Do we need/wait the `\n`?

File: dmoj/contrib/testlib.py [hunk 3]
```diff
@@ -14,7 +14,7 @@ class ContribModule(DefaultContribModule):
     PARTIAL = 7
 
     name = 'testlib'
-    repartial = re.compile(br'^points (\d+)\n$')
+    repartial = re.compile(br'^points (\d+)\n$', re.M)
```
Comment (quantum5) [inline dmoj/contrib/testlib.py:17]: Do we need/wait the `\n`?

File: dmoj/contrib/coci.py (hunk 1 above)
Comment (kiritofeng) [reply dmoj/contrib/coci.py:1]: Windows line ending nonsense.

File: dmoj/contrib/coci.py (hunk 2 above)
Comment (kiritofeng) [reply dmoj/contrib/coci.py:10]: IMO we should maintain parity with testlib.h, whatever that may be...

Also I would recommend against removing entirely, as then we'd have to modify every checker that uses partials...

File: dmoj/contrib/testlib.py (hunk 3 above)
Comment (kiritofeng) [reply dmoj/contrib/testlib.py:17]: IMO we should maintain parity with testlib.h, whatever that may be...

Also I would recommend against removing entirely, as then we'd have to modify every checker that uses partials...

Comment (quantum5) [reply dmoj/contrib/testlib.py:17]: No I mean the regex will match the same either way. Do we just want to not care if testlib outputs the trailling `\n` or not?

Comment (kiritofeng) [reply dmoj/contrib/testlib.py]: Decided to drop the trailing `\n`.

**Pull request merged**